tests/integration/helpers/s3_mocks/broken_s3.py

```diff
@@ -195,6 +195,18 @@ def inject_error(self, request_handler):
             )
             request_handler.write_error(429, data)
 
+    class TotalQpsLimitExceededAction:
```

Comment:
I'm guessing the reasoning comes from https://www.alibabacloud.com/help/en/oss/support/http-status-code-503 ? If so, it'd be nice to leave a comment, because we'll upgrade the aws module and this might be lost and nobody will know why it was introduced in the first place